README.rst

```diff
@@ -62,33 +62,101 @@ It is suggested to install a pre-commit hook in order to apply Black before push
-* SQL Alchemy 1.3
+    $ python3 --version
 
-For Postgres support and unit tests these packages are also required:
+If you don't have Python 3.7+ in your system, please download it from `python.org <https://www.python.org/downloads/>`_
 
-* geoalchemy2
-* psycopg2
-* nose2 (0.9.1)
+Pip is also necessary to create virtual environment. If you don't have it in your system, please download it::
 
-Dependencies
-------------
+    $ sudo apt-get install python3-pip
+    $ pip3 install virtualenv
 
-Install the python dependencies with: :code:`pip install -r requirements.txt`
+It is possible to verify pip installation with the following command::
+
+    $ pip3 --version
+
+----------
+For Ubuntu
+----------
+
+Installing Spatialite
+*********************
+1. Easy Way
+"""""""""""
+
+Open a terminal and run the following command::
+
+    $ sudo apt install spatialite-bin
+
+2. Alternative Way (Compiling From Source)
+""""""""""""""""""""""""""""""""""""""""""
+
+On Debian-based distributions the following libraries are necessary to run SpatiaLite::
+
+    $ sudo apt install zlib1g-dev libfreexl1
+    $ sudo apt-get install sqlite3 libsqlite3-dev
+    $ sudo apt-get install -y libsqlite3-mod-spatialite
+    $ sudo apt-get install binutils libproj-dev gdal-bin libgeos-dev libxml2-dev
+
+After all libraries are installed, it is necessary to download and run the latest SpatiaLite version::
+
+    $ wget https://www.gaia-gis.it/gaia-sins/libspatialite-sources/libspatialite-X.Y.Z.tar.gz
+    $ tar xaf libspatialite-X.Y.Z.tar.gz
+    $ cd libspatialite-X.Y.Z
+    $ ./configure
+    $ make
+    $ sudo make install
+
+Installing PostGIS
+******************
+
+The best way to install PostGIS is running the codes as follows::
+
+    sudo apt-get install libpq-dev python-dev
+    sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable
+    sudo apt-get update
+    sudo apt-get install postgis
+
+Cloning the Repository
+**********************
+Navigate to folder you would like to download the repository and run the following command::
+
+    $ git clone https://github.com/debrief/pepys-import.git
+
+Creating Python Environment
+***************************
+Virtual Environment might be used to run the project. For creating a proper one,
+the following commands must be executed respectively in your project directory (please change X with your Python version)::
+
+    $ virtualenv --python=python3.X venv
+
+**Note:** If you downloaded virtualenv in the current session, virtualenv command won't work,
+please try to run it after rebooting your machine.
+
+It is also possible to create a virtual environment with the following code::
+
+    $ python3.X -m virtualenv venv
+
+When environment is created successfully, please run the following commands::
+
+    $ source venv/bin/activate
+    $ pip install -r requirements.txt
 
 Unit tests
 ----------
 
+* In order to run the tests, please install requirements_dev: :code:`pip install -r requirements_dev.txt`
```

Comment:
Ok, when I run the unit tests in the next line, they fail with this error:
```
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) /usr/lib/x86_64-linux-gnu/mod_spatialite.so: cannot open shared object file: No such file or directory
```

I'm pretty sure this means we have to run line 109 above.

Reply:
Solved:

https://github.com/debrief/pepys-import/pull/66/commits/0cbbd2c0cd107a4de202c754b8e171068309d7bd